.github/workflows/deploy.yml

```
@@ -23,12 +23,29 @@ jobs:
         with:
           python-version: 3.9.12
       - id: stop-early
-        run: if "${GITHUB_WORKSPACE}/.github/has-functional-changes.sh" ; then echo "::set-output name=status::success" ; fi
+        run: |
+          if "${GITHUB_WORKSPACE}/.github/has-functional-changes.sh" ; then 
+            echo "status=success" >> $GITHUB_OUTPUT
```

If we go multiline, let's do it all the way 😉 
```suggestion
          if "${GITHUB_WORKSPACE}/.github/has-functional-changes.sh"
          then echo "status=success" >> $GITHUB_OUTPUT
```